orangewidget/gui.py

```diff
@@ -1458,9 +1461,12 @@ def comboBox(widget, master, value, box=None, label=None, labelWidth=None,
     else:
         hb = widget
 
-    combo = OrangeComboBox(
-        hb, maximumContentsLength=maximumContentsLength,
-        editable=editable)
+    if search:
+        combo = OrangeComboBoxSearch(hb)
```

Comment:
Does this mean that setting `seach=True` disables/ignores the `maximumContentsLength` and `editable` parameters? How do we feel about that?
Also this interaction is not mentioned anywhere!